metadata.rb

```diff
@@ -4,19 +4,16 @@
 license           'Apache 2.0'
 description       'Installs and configures haproxy'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '3.0.1'
-
-recipe 'haproxy', 'Installs and configures haproxy'
-recipe 'haproxy::app_lb', 'Installs and configures haproxy by searching for nodes of a particular role'
+version           '4.0.0'
+source_url        'https://github.com/sous-chefs/haproxy' if respond_to?(:source_url)
+issues_url        'https://github.com/sous-chefs/haproxy/issues' if respond_to?(:issues_url)
+chef_version      '>= 12.5' if respond_to?(:chef_version)
```

Comment:
You can probably avoid the respond_to portion since you're aiming to be a 12.5+ compatible cookbook.